.travis.yml

```diff
@@ -1,21 +1,10 @@
 language: node_js
 node_js: 'lts/*'
-cache:
-  directories:
-    - $HOME/.npm
 install:
-  - npm install -g npm
   - npm ci
 script:
-  - make checkformatting
-  - make lint
-  - npm test
-  - make SETTINGS_FILE=settings/chrome-prod.json dist/$(date +'%Y%m%d')-chrome-prod.zip
-  - make SETTINGS_FILE=settings/chrome-qa.json dist/$(date +'%Y%m%d')-chrome-qa.zip
-  - make SETTINGS_FILE=settings/firefox-prod.json dist/$(date +'%Y%m%d')-firefox-prod.xpi
-  - make SETTINGS_FILE=settings/firefox-qa.json dist/$(date +'%Y%m%d')-firefox-qa.xpi
-addons:
-  artifacts:
-    paths:
-      # Upload all built extension packages
-      - $(ls dist/*.zip dist/*.xpi | tr "\n" ":")
```

Comment:
We don't need the build artifacts so we can remove this step and also the code to give the packages unique names. It is still worth building them to make sure that step succeeds.

Reply:
Is this because it's jenkins' job to do the actual uploading, thus meaning that travis doesn't need these artifacts for anything?

Reply:
Exactly!